style.css

```
@@ -0,0 +1,147 @@
+body {
+  font-family: "Monserrat", sans-serif;
+  /* overflow: hidden; */
+  background-image: url("assets/bg2.png");
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  background-attachment: fixed;
+}
+:root {
+  --primary-colour: ;
+  --secondary-colour: ;
+  --text-colour: ;
+}
+
+form {
+  background: black;
+  text-align: center;
+}
+section {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding-left: 50px;
+}
+
+.header-image-container {
+  height: 100%;
+  padding: 20px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.header-image-container img {
+  height: 100px;
+  max-width: 100%;
+  padding: 20px;
+}
+
+.search-box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 400px;
+  padding: 40px;
+  transform: translate(-50%, -50%);
+  background: black;
+  box-sizing: border-box;
+  box-shadow: 0 15px 25px rgba(0, 0, 0, 0.6);
+  border-radius: 10px;
```

Remove positioning here. Just apply flex to the parent.
```suggestion
  width: 400px;
  padding: 40px;
  background: black;
  box-sizing: border-box;
  box-shadow: 0 15px 25px rgba(0, 0, 0, 0.6);
  border-radius: 10px;
```